packages/design-tokens/src/js/typography/types.ts

```diff
@@ -10,7 +10,6 @@ export type ThemeTypography = {
   sDisplayMD: TypeStyle;
   sHeadingLG: TypeStyle;
   sHeadingMD: TypeStyle;
-  sHeadingSMRegular: TypeStyle;
```

Comment:
Removing deprecated tokens from types. Ensured no instances left in codebase

<img width="493" alt="Screenshot 2025-05-25 at 2 33 22 PM" src="https://github.com/user-attachments/assets/b8d2fe71-cb8c-4949-a8e6-5d7d5c1a19c7" />